partition/uboot.go

```diff
@@ -49,6 +49,10 @@ func (u *uboot) Dir() string {
 	return filepath.Join(dirs.GlobalRootDir, "/boot/uboot")
 }
 
+func (u *uboot) ConfigFile() string {
+	return u.envFile()
+}
+
 func (u *uboot) envFile() string {
```

Comment:
Hmm.. I'm a bit confused actually. Looking at the grub implementation, it actually has an env and a conf file. Do we really mean to copy the env file out of the gadget?